Content.Server/Decals/DecalSystem.cs

```diff
@@ -314,21 +314,20 @@ public override void Update(float frameTime)
 
                     if (_dirtyChunks.TryGetValue(gridId, out var dirtyChunks))
                     {
-                        gridChunks.IntersectWith(dirtyChunks);
-                        newChunks.UnionWith(gridChunks);
+                        newChunks.UnionWith(dirtyChunks);
                     }
 
                     if (newChunks.Count == 0)
                         continue;
 
-                    updatedChunks[gridId] = newChunks;
+                    updatedChunks[gridId] = gridChunks;
```

Comment:
wait what? im p sure this is wrong, you are completely ignoring the dirtyChunks with this

Reply:
A.

The main issue is when we send only a dirty chunk it bulldozes the entire thing. I imagine the client should probably just cull its own chunks out of range and not care about the rest.